content/news/012.dd-ix-at-ripe89.de.md

```diff
@@ -0,0 +1,16 @@
+---
+published: 2024-11-06
+description: Marcel's Lightning Talk und viele gute Erinnerungen
+title: DD-IX auf RIPE89 in Prag
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens und Melody von DD-IX besuchten gemeinsam die RIPE-Community bei der diesjährigen Herbst-RIPE in Prag.
+Es war eine tolle Woche mit viel Spaß und interessanten Diskussionen über die Zukunft des Internets in Europa. Nachdem wir mit unserer IXP-from-Scratch Serie den RIPE Labs Artikelwettbewerb gewonnen hatten,
```

Comment:
Es war eine tolle Woche mit vielen interessanten Diskussionen über die Zukunft des Internets in Europa. Unser persönliches Higlight war der Vortrag von Marcel ;), der unsere RIPE Labs Serie "IXP-from-Scratch" vor mehr als 600 Teilnehmern präsentieren durfte. Der Vortrag ist [zum Nachschauen](https://ripe89.ripe.net/archives/video/1542/) verfügbar.